---
layout: page
title: Publications
permalink: /publications
---

For an up to date list of all publications (including preprints), please see [Google scholar](https://scholar.google.com/citations?user=H4jO_DQAAAAJ&hl=en).

[2023](#2023) [2022](#2022) [2021](#2021) [2020](#2020) [2019](#2019) [2018](#2018) [2017](#2017) [2016](#2016)

## <a id="2023"></a>2023

1. Ioannis Mouratidis, Candace S Y Chan, Nikol Chantzi, Georgios Christos Tsiatsianis, **Martin Hemberg**, Nadav Ahituv, Ilias Georgakopoulos-Soares, [*Quasi-prime peptides: identification of the shortest peptide sequences unique to a species*](https://academic.oup.com/nargab/article/5/2/lqad039/7138487), **NAR Genomics and Bioinformatics**, Volume 5, Issue 2, June 2023.

2. Kevin Riehl, Michael Neunteufel, **Martin Hemberg**, [*Hierarchical confusion matrix for classification performance evaluation*](https://academic.oup.com/jrsssc/advance-article-abstract/doi/10.1093/jrsssc/qlad057/7217007?redirectedFrom=fulltext), **Journal of the Royal Statistical Society Series C**, July 2023.

## <a id="2022"></a>2022

1. Kevin Riehl, Cristian Riccio, Eric A Miska, **Martin Hemberg**, [*TransposonUltimate: software for transposon classification, annotation and detection*](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac136/6541023), **Nucleic Acids Research**, 10.1093/nar/gkac136, 2022.

2. Ilias Georgakopoulos-Soares, Jesus Victorino, Guillermo E. Parada, Vikram Agarwal, Jingjing Zhao, Hei Yuen Wong, Mubarak Ishaq Umar, Orry Elor, Allan Muhwezi, Joon-Yong An, Stephan J. Sanders, Chun Kit Kwok, Fumitaka Inoue, **Martin Hemberg**, Nadav Ahituv, [*High-throughput characterization of the role of non-B DNA motifs on promoter function*](https://www.sciencedirect.com/science/article/pii/S2666979X22000313), **Cell Genomics**, 2, 2022.

3. Pilar Baldominos, Alex Barbera-Mourelle, Olga Barreiro, Yu Huang, Andrew Wight, Jae-Won Cho, Xi Zhao, Guillem Estivill, Isam Adam, Xavier Sanchez, Shannon McCarthy, Julien Schaller, Zara Khan, Albert Ruzo, Ricardo Pastorello, Edward T Richardson, Deborah Dillon, Paula Montero-Llopis, Romualdo Barroso-Sousa, Juliet Forman, Sachet A Shukla, Sara M Tolaney, Elizabeth A Mittendorf, Ulrich H von Andrian, Kai W Wucherpfennig, **Martin Hemberg**, Judith Agudo, [*Quiescent cancer cells resist T cell attack by forming an immunosuppressive niche*](https://www.sciencedirect.com/science/article/pii/S0092867422003439), **Cell**, 2022.

4. Ilias Georgakopoulos-Soares, Guillermo E Parada, Hei Yuen Wong, Ragini Medhi, Giulia Furlan, Roberto Munita, Eric A Miska, Chun Kit Kwok, **Martin Hemberg**, [*Alternative splicing modulation by G-quadruplexes*](https://www.nature.com/articles/s41467-022-30071-7), **Nature Communications**, 13(1), 2022.

5. Fredrik Salmen, Joachim De Jonghe, Tomasz S. Kaminski, Anna Alemany, Guillermo E. Parada, Joe Verity-Legg, Ayaka Yanagida, Timo N. Kohler, Nicholas Battich, Floris van den Brekel, Anna L. Ellermann, Alfonso Martinez Arias, Jennifer Nichols, **Martin Hemberg**, Florian Hollfelder, Alexander van Oudenaarden, [*High-throughput total RNA sequencing in single cells using VASA-seq*](https://www.nature.com/articles/s41587-022-01361-8), **Nature Biotechnology**, 2022.

6. Ilias Georgakopoulos-Soares, Guillermo E Parada, **Martin Hemberg**, [*Secondary structures in RNA synthesis, splicing and translation*](https://www.sciencedirect.com/science/article/pii/S2001037022001994), **Computational and Structural Biology Journal**, 20, p2771-2784, 2022.

7. Guillermo E Parada, **Martin Hemberg**, [*Bridge over troubled transcripts*](https://www.nature.com/articles/s43588-022-00219-y), **Nature Computational Science**, 2, p142-143, 2022.

8.  Kathryn J Gray, **Martin Hemberg**, S Ananth Karumanchi, [*Cell-Free RNA Transcriptome and Prediction of Adverse Pregnancy Outcomes*](https://academic.oup.com/clinchem/advance-article-abstract/doi/10.1093/clinchem/hvac109/6656108), **Clinical Chemistry**, 2022.

9. Ilias Georgakopoulos-Soares, Candace S. Y. Chan, Nadav Ahituv, **Martin Hemberg**, [*High-throughput techniques enable advances in the roles of DNA and RNA secondary structures in transcriptional and post-transcriptional gene regulation*](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02727-6), **Genome Biology**, 23, 2022. 

10. Guillermo E Parada, **Martin Hemberg**, [*Identification and Quantification of Microexons using Bulk and Single-Cell RNA-Seq Data*](https://link.springer.com/protocol/10.1007/978-1-0716-2521-7_8), [**Alternative Splicing**](https://link.springer.com/book/10.1007/978-1-0716-2521-7), 2022.

11. Tobias Bergmann, Yong Liu, Jonathan Skov, Leo Mogus, Julie Lee, Ulrich Pfisterer, Louis-Francois Handfield, Andrea Asenjo-Martinez, Irene Lisa-Vargas, Stefan E Seemann, Jimmy Tsz Hang Lee, Nikolaos Patikas, Birgitte Rahbek Kornum, Mark Denham, Poul Hyttel, Menno P Witter, Jan Gorodkin, Tune H Pers, **Martin Hemberg**, Konstantin Khodosevich, Vanessa Jane Hall, [*Production of human enthorinal stellate cell-like cells by forward programming shows an important role of Foxp1 in reprogramming*](https://www.frontiersin.org/articles/10.3389/fcell.2022.976549/full?field=&journalName=Frontiers_in_Cell_and_Developmental_Biology&id=976549), **Frontiers in Cell and Developmental Biology**, 2022. 

12.  Junha Cha, Jiwon Yu, Jae-Won Cho, **Martin Hemberg**, Insuk Lee, [*scHumanNet: a single-cell network analysis platform for the study of cell-type specificity of disease genes*](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac1042/6814446), **Nucleic Acids Research**, 2022.

13. Fu Xian Quah, **Martin Hemberg**, [*SC3s: efficient scaling of single cell consensus clustering to millions of cells*](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-05085-z), **BMC Bioinformatics**, 2022.

## <a id="2021"></a>2021

1. Guillermo E Parada, Roberto Munita, Ilias Georgakopoulos-Soares, Hugo JR Fernandes, Veronika R Kedlian, Emmanouil Metzakopian, Maria Estela Andres, Eric A Miska, **Martin Hemberg**, [*MicroExonator enables systematic discovery and quantification of microexons across mouse embryonic development*](https://link.springer.com/article/10.1186/s13059-020-02246-2), **Genome Biology** 22(1) p1-26, 2021. 

2.  Katharina Gapp, Guillermo Parada, Fridolin Gross, Alberto Corcoba, Jasmine Kaur, Evelyn Grau, **Martin Hemberg**, Johannes Bohacek, Eric A Miska, [*Single paternal Dexamethasone challenge programs offspring metabolism and reveals multiple candidates in RNA-mediated inheritance*](https://www.sciencedirect.com/science/article/pii/S2589004221008385), **iScience**, 2021.

3.  Guste Urbonaite, Jimmy Tsz Hang Lee, Ping Liu, Guillermo E. Parada, **Martin Hemberg**, Murat Acar, [*A yeast-optimized single-cell transcriptomics platform elucidates how mycophenolic acid and guanine alter global mRNA levels*](https://www.nature.com/articles/s42003-021-02320-w), **Communications Biology**, 4:822, 2021.

4.  Svenja Schüler, Joanna Kirkpatrick, Manuel Schmidt, Deolinda Santinha, **Martin Hemberg**, Alessandro Ori, Julia von Maltzahn, [*Deep proteome of aging skeletal muscles reveals Smoc2 as a modulator of satellite cell regenerative capacity*](https://www.sciencedirect.com/science/article/pii/S221112472100574X), **Cell Reports**, 35(10), 2021.

5.  Jimmy Tsz Hang Lee, Nikolaos Patikas, Vladimir Yu Kiselev, **Martin Hemberg**, [*Fast searches of large collections of single cell data using scfind*](https://www.nature.com/articles/s41592-021-01076-9), **Nature Methods** 18(3), p262-271, 2021.

6.  Ilias Georgakopoulos-Soares, Ioannis Mouratidis, Guillermo E. Parada, Navneet Matharu, **Martin Hemberg**, Nadav Ahituv, [*Asymmetron: A toolkit for the identification of strand asymmetry patterns in biological sequences*](https://academic.oup.com/nar/article-abstract/49/1/e4/5992281), **Nucleic Acids Research** 49(1), 2021.

7.  Ruben Chazarra-Gil, Stijn van Dongen, Vladimir Kiselev, **Martin Hemberg**, [*Flexible comparison of batch correction methods for single-cell RNA-seq using BatchBench*](https://academic.oup.com/nar/article-abstract/49/7/e42/6125660), **Nucleic Acids Research** 49(7), 2021.

8. Patrick Cahan, Davide Cacchiarelli, Sara-Jane Dunn, **Martin Hemberg**, Susana M. Chuva de Sousa Lopes, Samantha Morris, Owen Rackham, Antonio del Sol, Christine A. Wells, *Computational Stem Cell Biology: open questions and guiding principles*, **Cell Stem Cell** 28(1), p20-32, 2021.

9. Tallulah S Andrews, Vladimir Kiselev, Davis McCarthy, **Martin Hemberg**, [*Computational analysis of single cell RNA-seq: an overview*](https://www.nature.com/articles/s41596-020-00409-w), **Nature Protocols** 16(1), p1-9, 2021.

10. Ilias Georgakopoulos-Soares, Ofer Yizhar-Barnea, Ioannis Mouratidis, **Martin Hemberg**, Nadav Ahituv, [*Absent from DNA and protein: genomic characterization of nullomers and nullpeptides across functional categories and evolution*](https://link.springer.com/article/10.1186/s13059-021-02459-z), **Genome Biology**, 22:1, 2021.

## <a id="2020"></a>2020

1. Jennifer Westoby, Pavel Artemov, **Martin Hemberg**, Anne Ferguson-Smith, [*Obstacles to Studying Alternative Splicing Using scRNA-seq*](https://link.springer.com/article/10.1186/s13059-020-01981-w), **Genome Biology** 21(1), p1-19, 2020.

2.  Haynes Heaton, Arthur M. Talman, Andrew Knights, Maria Imaz, Richard Durbin, **Martin Hemberg**, Mara Lawniczak, [*souporcell: Robust clustering of single cell RNAseq by genotype and ambient RNA inference without reference genotypes*](https://www.nature.com/articles/s41592-020-0820-1), **Nature Methods**, 17(6), p615-620, 2020.

3.  Rong Wang, Ashley Laughney, Roshan Sharma, Xiaojuan Shen*, Kosuke Funato, Phil Clark, Monika Shpokayte, Peter Morgenstern, Monalisa Navare, Yiqi Xu, Shaghayegh Harbi, Ignas Masilionis, Gouri Nanjangud, **Martin Hemberg**, Dana Pe’er, Viviane Tabar, [*Dormant Radial Glia like cells in Adult Human Brain Tumor of Cortex*](https://www.sciencedirect.com/science/article/pii/S221367112030028X), **Stem Cell Reports**, 14(2), 338-350, 2020.

4.  Ilias Georgakopoulos-Soares, Gene Koh, Josef Jiricny, **Martin Hemberg**, Serena Nik-Zainal, [*Transcription-coupled repair and mismatch repair contribute towards preserving genome integrity at mononucleotide repeat tracts*](https://www.nature.com/articles/s41467-020-15901-w), **Nature Communications**, 11(1), 2020.

5.  Omer Ali Bayraktar, Theresa Bartels, Staffan Holmqvist, Vitalii Kleshchevnikov, Araks Martirosyan, Damon Polioudakis, Lucile Ben Haim, Adam MH Young, Mykhailo Y Batiuk, Kirti Prakash, Alexander Brown, Kenny Roberts, Mercedes F Paredes, Riki Kawaguchi, John H Stockley, Khalida Sabeur, Sandra M Chang, Eric Huang, Peter Hutchinson, Erik M Ullian, **Martin Hemberg**, Giovanni Coppola, Matthew G Holt, Daniel H Geschwind, David H Rowitch, [*Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map*](https://www.nature.com/articles/s41593-020-0602-1), **Nature Neuroscience** 23(4), p500-509, 2020.

## <a id="2019"></a>2019

1. Virginia M. Howick, Andrew J. C. Russell, Tallulah S Andrews, Haynes Heaton, Adam J Reid, Kedar Natarajan, Hellen Butungi,Tom Metcalf, Lisa H. Verzier,  Julian C. Rayner, Matthew Berriman, Jeremy K. Herren, Oliver Billker, **Martin Hemberg**, Arthur M. Talman, Mara K.N. Lawniczak, [*The Malaria Cell Atlas: a comprehensive reference of individual parasite transcriptomes across the complete Plasmodium life cycle*](https://science.sciencemag.org/content/365/6455/eaaw2619.abstract), **Science** 365(6455), 2019.

2. Tallulah S Andrews, **Martin Hemberg**, [*M3Drop: dropout-based feature selection for scRNASeq*](https://academic.oup.com/bioinformatics/article-abstract/35/16/2865/5258099), **Bioinformatics** 35(16), p2865-67, 2019.

3. Vladimir Y Kiselev, Tallulah S Andrews, **Martin Hemberg**, *Challenges for unsupervised clustering of scRNA-seq data*, **Nature Review Genetics**, 20(5), p273-282, 2019.

4. Jimmy Tsz Hang Lee, **Martin Hemberg**, *Supervised clustering of single cell data* (News & Views), **Nature Methods** 16(10), p965-66, 2019.

5. Tallulah S Andrews, Vladimir Y Kiselev, **Martin Hemberg**, *Statistical methods for scRNA-seq*, chapter for **Handbook of Statistical Genetics**, 4th edition, Wiley, 2019.

## <a id="2018"></a>2018

1. Jennifer Westoby, Marcela Sjoberg, Anne C Ferguson-Smith, **Martin Hemberg**, [*Simulation based benchmarking of isoform quantification in single-cell RNA-seq*](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1571-5), **Genome Biology**, 19(1), p1-14, 2018.

2. Tallulah S Andrews, **Martin Hemberg**, [*False signals induced by single-cell imputations*](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6415334.2/), **F1000 Research**, 7:1740, 2019.

3. Ilias Georgakopoulos-Soares, Sandro Morganella, Naman Jain, **Martin Hemberg**, Serena Nik-Zainal, [*Non-canonical secondary structures arising from non-B-DNA motifs are primary determinants of mutagenesis*](https://genome.cshlp.org/content/28/9/1264.short), **Genome Research**, 28(9), p1264-1271, 2018.

4. Vladimir Y Kiselev, Andrew Yiu, **Martin Hemberg**, [*scmap - a tool projection of single-cell RNA-seq data onto a reference*](https://www.nature.com/articles/nmeth.4644), **Nature Methods**, 15(5), p359-362, 2018. 

5. Isabelle Bergiers, Tallulah S Andrews, Ozge Vargel, Andreas Buness, Ewa Janosz, Natalia Lopez-Anguita, Kerstin Ganter, Kinga Kosim, Cemre Celen, Gulce Perçin, Paul Collier, Bianka Baying, Vladimir Benes, **Martin Hemberg**, Christophe Lancrin, [*Single cell transcriptomics reveals new insights on the dynamical function of transcription factors during blood stem and progenitor cell formation*](https://elifesciences.org/articles/29312), **eLife**, 7:e29312, 2018.

6. Paulo P Amaral, Tommaso Leonardi, Namshik Han, Emmanuelle Vire, Dennis K Gascoigne, Raul Arias-Carrasco, Magdalena Buscher, Anda Zhang, Stefano Pluchino, Vinicius Maracaja-Coutinho, Helder I Nakaya, **Martin Hemberg**, Ramin Shiekhattar, Anton J Enright, Tony Kouzarides, [*Genomic positional conservation identifies topological anchor point (tap) RNAs linked to developmental loci*](https://link.springer.com/article/10.1186/s13059-018-1405-5), **Genome Biology**, 19:32, 2018.


## <a id="2017"></a>2017

1. Hansruedi Mathys, Chinnakkaruppan Adaikkan, Fan Gao, Jennie Z Young, Elodie Manet, **Martin Hemberg**, Philip L De Jager, Richard M Ransohoff, Aviv Regev, Li-Huei Tsai, [*Temporal Tracking of Microglia Activation in Neurodegeneration at Single-Cell Resolution*](https://www.sciencedirect.com/science/article/pii/S2211124717313141), **Cell Reports** 21(2), 366-380, 2017.

2. Aviv Regev, Sarah A. Teichmann, Eric S. Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, Peter Campbell, Piero Carninci, Menna Clatworthy, Hans Clevers, Bart Deplancke, Ian Dunham, James Eberwine, Roland Eils, Wolfgang Enard, Andrew Farmer, Lars Fugger, Berthold Gottgens, Nir Hacohen, Muzlifah Haniffa, **Martin Hemberg**, Seung Kim, Paul Klenerman, Arnold Kriegstein, Ed Lein, Sten Linnarsson, Joakim Lundeberg, Partha Majumder, John C. Marioni, Miriam Merad, Musa Mhlanga, Martijn Nawijn, Mihai Netea, Garry Nolan, Dana Pe’er, Anthony Phillipakis, Chris P. Ponting, Steve Quake, Wolf Reik, Orit Rozenblatt-Rosen, Joshua Sanes, Rahul Satija, Ton N Schumacher, Alex Shalek, Ehud Shapiro, Padmanee Sharma, Jay W. Shin, Oliver Stegle, Michael Stratton, Michael JT Stubbington, Alexander van Oudenaarden, Allon Wagner, Fiona Watt, Jonathan Weissman, Barbara Wold, Ramnik Xavier, Nir Yosef and the Human Cell Atlas Meeting Participants, [*The Human Cell Atlas*](https://elifesciences.org/articles/27041), **eLife** 6, 2017.

3. Alper Akay, Tomas Di Domenico, Kin Man Suen, Amena Nabih, Guillermo E Parada, Mark Larance, Ragini Medhi, Ahmet Can Berkyurek, Christopher J Wedeles, Xinlian Zhang, Ping Ma, Angus I Lamond, **Martin Hemberg**, Julie M Claycomb, Eric A Miska, [*The Aquarius/EMB-4 helicase licenses co-transcriptional gene silencing*](https://www.sciencedirect.com/science/article/pii/S1534580717305440), **Developmental Cell** 42(3), 241-255, 2017.

4. Kristina Kirschner, Tamir Chandra, Vladimir Y Kiselev, David Flores-Santa Cruz, Iain C Macaulay, Hyun Jun Park, Juan Li, David G Kent, Rupa Kumar, Dean C Pask, Tina L Hamilton, **Martin Hemberg**, Wolf Reik, Anthony R Green, [*Proliferation Drives Aging-Related Functional Decline in a Subpopulation of the Hematopoietic Stem Cell Compartment*](https://www.sciencedirect.com/science/article/pii/S2211124717306034), **Cell Reports** 19(8), 1503-1511, 2017.

5. Vladimir Y Kiselev, Kristina Kirschner, Michael T Schaub, Tallulah S Andrews, Andrew Yiu, Tamir Chandra, Kedar N Natarajan, Wolf Reik, Mauricio Barahona, Anthony R Green, **Martin Hemberg**, [*SC3-consensus clustering of single-cell RNA-Seq data*](https://www.nature.com/articles/nmeth.4236), **Nature Methods** 14(5), 483-486, 2017.


## <a id="2016"></a>2016

1. Ilias Georgakopoulos-Soares, Naman Jain, Jesse Gray, **Martin Hemberg**, [*MPRAnator: a web-based tool for the design of Massively Parallel Reporter Assay experiments*](https://academic.oup.com/bioinformatics/article-abstract/33/1/137/2525693), **Bioinformatics** 33(1), 137-138, 2016.

2. Andreas T Sorensen, Yonatan A Cooper, Michael V Baratta, Feng-Ju Weng, Yuxiang Zhang, Kartik Ramamoorthi, Robin Fropf, Emily LaVerriere, Jian Xue, Andrew Young, Colleen Schneider, Casper Rene Gotzsche, **Martin Hemberg**, Jerry CP Yin, Steven F Maier, Yingxi Lin, [*A robust activity marking system for exploring active neuronal ensembles*](https://elifesciences.org/articles/13918), **eLife** 5, e13918, 2016.

3. Thomas A. Nguyen, Richard D. Jones, Andrew Snavely, Andreas Pfenning, Rory Kirchner, **Martin Hemberg**, Jesse M. Gray, *High-throughput functional comparison of promoter and enhancer activities*, **Genome Research** 26(8), 1023-33, 2016.

4. Yue  Yang,  Tomoko  Yamada,  Kelly  K.  Hill,  **Martin  Hemberg**,  Naveen  Reddy,  Ha  Y. Cho, Arden Guthrie, Anna Oldenborg, Shane A. Heiney, Shogo Ohmae, Javier F. Medina, Timothy E. Holy, Azad Bonni, *Chromatin Remodeling Shuts Off Activity Genes to Regulate Dendrite Elimination and Sensorimotor Encoding*, **Science** 353(6296), 300-5, 2016.

5. Mihails Delmans, **Martin Hemberg**, [*Discrete distributional differential expression (D3E) - a tool for gene expression analysis of single-cell RNA-seq data*](https://link.springer.com/article/10.1186/s12859-016-0944-6), **BMC Bioinformatics**, 17(1), 2016.


